src/syntheticdata-mcp-server/pyproject.toml

```
@@ -0,0 +1,119 @@
+[project]
+name = "awslabs.syntheticdata-mcp-server"
+version = "0.0.0"
+description = "An AWS Labs Model Context Protocol (MCP) server for syntheticdata"
+readme = "README.md"
+requires-python = ">=3.10"
+dependencies = [
+    "mcp[cli]>=1.6.0",
+    "pydantic>=2.10.6",
+    "pandas>=2.0.0",
+    "boto3>=1.34.0",
+]
+license = {text = "Apache-2.0"}
+license-files = ["LICENSE", "NOTICE" ]
+authors = [
+    {name = "Amazon Web Services"},
+    {name = "AWSLabs MCP", email="203918161+awslabs-mcp@users.noreply.github.com"},
+]
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+
+[project.urls]
+homepage = "https://awslabs.github.io/mcp/"
+docs = "https://awslabs.github.io/mcp/servers/syntheticdata-mcp-server/"
+documentation = "https://awslabs.github.io/mcp/servers/syntheticdata-mcp-server/"
+repository = "https://github.com/awslabs/mcp.git"
+changelog = "https://github.com/awslabs/mcp/blob/main/src/syntheticdata-mcp-server/CHANGELOG.md"
+
+[project.scripts]
+"awslabs.syntheticdata-mcp-server" = "awslabs.syntheticdata_mcp_server.server:main"
+
+[dependency-groups]
+dev = [
+    "commitizen>=4.2.2",
+    "pre-commit>=4.1.0",
+    "ruff>=0.9.7",
+    "pyright>=1.1.398",
+    "pyright>=1.1.398",
```

Duplicated package, add pytest-cov 
```suggestion
    "pytest-cov>=4.1.0",
```

---

Done